src/algebra/group_with_zero.lean

```diff
@@ -371,6 +377,21 @@ classical.by_contradiction $ λ ha, h₁ $ mul_right_cancel' ha $ h₂.symm ▸
 
 end cancel_monoid_with_zero
 
+section comm_cancel_monoid_with_zero
+
+variables [cancel_monoid_with_zero M₀] {a b c : M₀}
```

Comment:
This section / the definitions are called `comm_cancel_monoid_with_zero`, but the assumption is only `cancel_monoid_with_zero`. Could you make them consistent?